### PR TITLE
Changed structure of this document to a more understandable document

### LINKED ISSUE #32: Updating documentation
upcoming changes + made changes need to be documented for other group.

### PR DESCRIPTION
**Pull request by mspanoghe**
Whole new structure. Better?
closes #32

### CODE REVIEW

**Comment by RolfStarre** [comment]
In 2.1 I would give a short description of what the utility is.

Structure is good.

**Comment by mspanoghe** [comment]
Good point. Added the information

File: documents/EmotionConfig Documentation.tex [hunk 1]
```diff
@@ -31,58 +31,149 @@
...
+	\hline  
+\end{tabular}
+\\
+
+\subsection{Goal definition}
+\subsubsection{With utility}
+This definition defines the properties of a goal. Agent is the agent for which this specification is true (this is not yet implemented the idea is that eventually you could define different properties for different agents but for the same goal. At the moment it can be filled in with anything and the specification will hold for all agents.). Goal is the name of the goal, this should the same name as the goal is given in GOAL including the arity. Defining a goal like this also adds it to the whitelist if this feature is being used.\\
+\begin{tabular}{|l|l|l|l|l|}
+	\hline  Types& String & String & String & double \\ 
+	\hline  Command & GOAL & agentName & goalName/arity & utility  \\
```
Comment (berndkr) [inline documents/EmotionConfig Documentation.tex:138]: Goals hebben nu geen agent names meer

**Comment by mspanoghe** [comment]
Mooi!

**Pull request merged**